include/deal.II/multigrid/mg_constrained_dofs.h

```diff
@@ -383,14 +393,22 @@ MGConstrainedDoFs::have_boundary_indices() const
 
 
 inline const AffineConstraints<double> &
-MGConstrainedDoFs::get_level_constraint_matrix(const unsigned int level) const
+MGConstrainedDoFs::get_level_constraints(const unsigned int level) const
 {
   AssertIndexRange(level, level_constraints.size());
   return level_constraints[level];
 }
 
 
 
+inline const AffineConstraints<double> &
+MGConstrainedDoFs::get_level_constraint_matrix(const unsigned int level) const
```

Comment:
Are we not planning on deprecating this function?

Reply:
I'm not sure I understand what you are asking. I put `DEAL_II_DEPRECATED` at the declaration. Is there something else that needs to be done here?